pkgs/os-specific/linux/mxu11x0/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, kernel }:
+
+assert stdenv.isLinux;
```

Comment:
I wonder if this is not somewhat redundant; I think `meta.platforms` will cause the build to fail on non-Linux systems already.